src/Compilers/Core/Portable/DiagnosticAnalyzer/CachingSemanticModelProvider.cs

```diff
@@ -23,30 +23,31 @@ namespace Microsoft.CodeAnalysis.Diagnostics
     /// </summary>
     internal sealed class CachingSemanticModelProvider : SemanticModelProvider
     {
+        public static CachingSemanticModelProvider Instance { get; } = new CachingSemanticModelProvider();
```

Comment:
Consider adding a comment linking to the CWT issue so someone doesn't unintentionally undo this change in the future.